### DISCUSSION TITLE
Image datatype doesn't let me choose from image manager

### DISCUSSION THREAD

**Discussion by fiammybe** [ImpressCMS/impresscms#1024] [Ideas]
When I define a datatype 'image' for a field, I get options to upload a new image, or to define a URL. I have no possibility to re-use an image that has been uploaded into the image manager, and that's really a lack of functionality.

**Comment by MekDrop** [comment]
Is this about 1.3 or 2.0?

**Comment by fiammybe** [comment]
Both. I think handling images should all be done via the image manager: upload new image, use an existing image, change a previously chosen image, offer a list of images based on criteria (part of a certain folder, only certain sizes,...)
